styleguide.config.js

```diff
@@ -96,43 +96,18 @@ module.exports = {
     TableRenderer: path.join(__dirname, 'styleguide/components/table'),
   },
   styleguideDir: 'build',
-  styles: {
-    Para: {
-      para: {
-        fontSize: 14,
-      },
-    },
-  },
+  styles: path.resolve(__dirname, './styleguide.config.styles.js'),
   template: {
     favicon: 'favicon.ico',
     head: {
       links: [
         {
           rel: 'stylesheet',
-          href: 'https://fonts.googleapis.com/css?family=Merriweather|Open+Sans',
+          href: 'https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap',
```

Comment:
While we can set up MDS to process its own fonts, this does not work in our bigmaven codebase. Looks like our bigmaven Webpack configuration runs out of memory processing the fonts -- I think it's more about SASS and outdated dependencies but cest la vie.